docs/_guide/templates.md

```diff
@@ -496,3 +496,14 @@ render() { return html`<slot name="thing"></slot>`; }
   <p slot="thing">stuff</p>
 </my-element>
 ```
+
+## Advanced features
```

Comment:
Rather than "Advanced Features", what about calling this section "Further Reading" and lead with a link to the lit-html site?

cc @katejeffreys

Reply:
I do agree with the idea of it being not advanced features, more a further reading. I made some minor changes based on your ideas, especially made the links stand out more. The wording of the list could be improved though.